src/base/bittorrent/session.cpp

```diff
@@ -321,6 +321,11 @@ QString Session::tempPath() const
     return m_tempPath;
 }
 
+QString Session::torrentTempPath(const InfoHash &hash) const
+{
+    return m_tempPath + QString(hash) + "/";
```

Comment:
So I understood you incorrectly. I thought you mean that we don't get unique path here: `temp_path/<torrent_hash>`.